### ISSUE TITLE
Odd behavior when running Overtone in Cider repl - print out of large meta data map

### ISSUE THREAD

**Issue by JulioBarros** [clojure-emacs/cider#2867]
So, I have a strange, to me, problem. Following the Overtone quick start instructions everything seems to work fine in a leiningen repl.

If I run the making sounds commands https://github.com/overtone/overtone/

```
user=> (use 'overtone.live)
--> Loading Overtone...

user=> (demo (sin-osc))
#<synth-node[loading]: user/audition-synth 33>

```

However, when I run it inside of a Cider repl, I get a large dict of meta data (?). I don't even know what it is so don't know how to search for a solution.

```
user=> (demo (sin-osc))
{:synth "insane-noisesabd/audition-synth",
 :id 34,
 :target 7,
 :position :tail,
 :args {},

...

 :status #<Atom@a2e76ba: :destroyed>,
 :loaded? #<Promise@4f2cf99b: true>}

```

It only seems to be a problem when I run Overtone in Cider. Other code prints fine and Overtone behaves in a Clojure/lein repl. So I don't know if it is an Overtone issue or a cider issue or something else.

Do I have a weird setting set somewhere? Anybody have any hints that may help me figure out what's going on?  

Thanks.

**Comment by bbatsov** [comment]
I'm guessing the difference comes because CIDER uses pretty-printing in the REPL by default. You can disable it and you should guess the same results as in `lein repl`. See https://docs.cider.mx/cider/0.25/repl/configuration.html#pretty-printing-in-the-repl for details.

**Comment by JulioBarros** [comment]
Thank you for the reply Bozhidar. Unfortunately it did not seem to change anything. I ran the emacs command you gave, re-ran the overtone function and still got a large map printout. I then did a describe-variable and it said:

```
cider-repl-use-pretty-printing is a variable defined in ‘cider-repl.el’.
Its value is nil
Original value was t
```

My Clojure is rusty but it looks like it might be printing the value of var instead of just the name. Would pretty printing generally dereference the var?  Thanks again.
